src/axolotl/utils/models.py

```diff
@@ -394,10 +394,14 @@ def apply_patches(self) -> None:
             and self.cfg.flash_attention
             and self.cfg.sample_packing
         ):
+            has_remote_code = (
+                "auto_map" in self.model_config
+                and self.model_type in self.model_config["auto_map"]
```

Comment:
Yeah I am not sure myself, the model above is labelled text classification model rather than text generation. The modeling file has `Qwen2ForCausalLM` but not `_get_unpad_data`
Will need to think how to tackle this

Reply:
Oh, that's my bad. I was just trying to find an example model which still uses remote code.